src/scss/_primary_nav.scss

```diff
@@ -27,23 +27,24 @@
 		}
 	}
 	.o-header-services__nav-link {
```

Comment:
I know its not part of this PR, but there's no newline between this rule and the rule above. All the rest of the declarations have that so this should probably be consistent.